### PR TITLE
Adds FAQ question to introduce library; fixes #193

### LINKED ISSUE #193: Add a mention of our library to the website
It would list the magazines we subscribe to, any books (electronic or physical) that are available to read. Not sure if this should be a brand new tab on the website or we should link to it in our about page?

### PR DESCRIPTION
**Pull request by zarinaaa**

### CODE REVIEW

File: content/pages/faq.md [hunk 1]
```diff
@@ -34,6 +34,13 @@ Our women-only events are not publicly advertised and we always specify
 Please see the **Volunteer Opportunities** section on our [About
 page]({filename}/pages/about.md)!
 
+## Do you have resources to learn more about women in tech? ##
+
+We have a library with feminist resources that are available for you to read
+during office hours. They include:
+* Quarterly publications from Model View Culture
```
Comment (evykassirer) [inline content/pages/faq.md:41]: oops - didn't bullet point properly. Pretty sure you want an extra line before this one.

you can see here: https://wics-site-pr-217.herokuapp.com/faq/

File: content/pages/faq.md [hunk 2]
```diff
@@ -34,6 +34,17 @@ Our women-only events are not publicly advertised and we always specify
...
 page]({filename}/pages/about.md)!
 
+## Do you have resources to learn more about women in tech? ##
+
+We have a library with feminist resources that are available for you to read
+during office hours. They include:
+<<<<<<< HEAD
+
+=======
+>>>>>>> 3f6266882d1a45d25b0c5293e30e1e7fb8fe7fae
```
Comment (arshiamufti) [inline content/pages/faq.md]: You might want to remove this merge conflict message :)

Comment (zarinaaa) [inline content/pages/faq.md]: I'll consider it.

File: content/pages/faq.md [hunk 3]
```diff
@@ -34,6 +34,14 @@ Our women-only events are not publicly advertised and we always specify
...
 page]({filename}/pages/about.md)!
 
+## Do you have resources to learn more about women in tech? ##
+
+We have a library with feminist resources that are available for you to read
+during office hours. They include:
+
+* Quarterly publications from Model View Culture 
+* Your Startup is Broken - Collected Essays by Shelley Kane
+
```
Comment (arshiamufti) [inline content/pages/faq.md:46]: We also have The Recompiler.

File: content/pages/faq.md [hunk 4]
```diff
@@ -34,6 +34,14 @@ Our women-only events are not publicly advertised and we always specify
 Please see the **Volunteer Opportunities** section on our [About
 page]({filename}/pages/about.md)!
 
+## Do you have resources to learn more about women in tech? ##
+
+We have a library with feminist resources that are available for you to read
+during office hours. They include:
+
+* Quarterly publications from Model View Culture 
+* Your Startup is Broken - Collected Essays by Shelley Kane
```
Comment (arshiamufti) [inline content/pages/faq.md]: lol this is spelt as "Shanley"

**Comment by bdamerac** [comment]
There is no space between the beginning of "I have a food sensitivity. Can you accomodate me at catered events?" and the end of the previous question's content.

File: content/pages/faq.md [hunk 5]
```diff
@@ -34,6 +34,16 @@ Our women-only events are not publicly advertised and we always specify
...
 
+## Do you have resources to learn more about women in tech? ##
+
+We have a library with feminist resources including:
+
+* Quarterly publications from Model View Culture 
+* Your Startup is Broken - Collected Essays by Shanley Kane
+* The Recompiler
+
+They are available to read during office hours.
```
Comment (evykassirer) [inline content/pages/faq.md:45]: solid hack 👍 
probably best to just do this until #161 is fixed

**Comment by fboxwala** [comment]
So as per #wics-sys you know you need to clean up the commit history of your branch a little. There are steps to sqaushing a commit here: https://github.com/wics-uw/git-workshop-W16/wiki/Squashing-Commits

Ping me if you need help!

**Comment by evykassirer** [comment]
looks good to me 🎉 

thanks @arshiamufti @Bhargavi1997 @fboxwala and @zarinaaa  😄 

@SrishtiG how do you feel about taking on the deploy for this one? I know you haven't gotten a chance to do one yet.

**Pull request merged**